usaspending_api/api_docs/api_documentation/advanced_award_search/transaction_spending_summary.md

```diff
@@ -0,0 +1,81 @@
+## [Transaction Spending Summary](#transaction-spending-summary)
+**Route:** `/api/v2/search/transaction_spending_summary/`
+
+**Method:** `POST`
+
+This route takes award filters, and returns the number of transactions and summation of federal action obligations.
+
+MVP is to only surface prime awards. Later on sub-awards will be added
```

Comment:
Change this whole summary to:

This route returns the number of transactions and the sum of federal action obligations for prime awards given a set of award of filters.


Remove the comments about MVP and future-facing commentary.

Reply:
done